spec/forms/work_form_spec.rb

```diff
@@ -53,12 +54,37 @@
     end
   end
 
+  describe 'contributors validation' do
+    let(:contributor_error) { { contributors: ['Please add at least one contributor.'] } }
+
+    context 'with no contributors' do
+      it 'does not validate' do
+        form.valid?
+        expect(form.errors.messages).to include(contributor_error)
+      end
+    end
+
+    context 'with contributors' do
+      let(:form_with_contributors) { described_class.new(work_with_contributors) }
+
+      it 'validates' do
+        form_with_contributors.valid?
```

Comment:
same question as above re: `expect(form).to be_valid`

Reply:
yes, had to build a more complete work object to actually make it valid in order to do this (else it would not be valid even with contributors), but still looks better